### ISSUE TITLE
a11y: "Back to Top" tab index, focus-highlight, and focus should shift to the top of the doc

### ISSUE THREAD

**Issue by ambersdai** [openstax/cnx#117]
## Overview
From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

### Issue 5: "Back to top" link is not accessible
This link does not have accessible function and is not presented in a meaningful sequence (1.3.2) and Focus Order (2.4.3).
Example (focus is still on this link, not on the top of the page). The following video depict this issue:
https://drive.google.com/file/d/0B2fFZC6BqvTLSTJEcHdzMFFtSzA/view?usp=sharing
### Reference to standards
1.3.2 Meaningful Sequence. When the sequence in which content is presented affects its meaning, a correct reading sequence can be programmatically determined. (Level A)
2.4.3 Focus Order. If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability. (Level A).
### Suggestion
Make this link accessible: after pressing "back to top" the page is scrolled to top and focus is on the first active element of the page or on the first element of the main area.

## PRs
What PRs need to be merged to complete this task/feature
## Test Plan
How will QA know this feature works? What parts of regression tests would be most affected?
## Acceptance Criteria
Expected behavior
